pkg/alertmanager/amcfg.go

```diff
@@ -121,6 +122,22 @@ func (cg *configGenerator) generateConfig(
 	// alerts will fallthrough.
 	baseConfig.Route.Routes = append(subRoutes, baseConfig.Route.Routes...)
 
+	// set default slack url
+	globalSlackAPIURL, err := url.Parse("http://slack.example.com")
```

Comment:
Why set it by default to example string? IMHO this can lead to silent issues after installation.

Reply:
If I'm understanding this correctly it's because when creating the CRD for the alertmanager config it doesn't allow for the creation of the 'Global' section [here](https://docs.okd.io/latest/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html). The slack webhook URL is defined on an individual basis via the `slackConfigs` section of the alert but still needs to be present in the Global block in order to be valid which the operator generates when it converts that CRD into the secret.

I do agree though that if slack integration isn't being used then you probably wouldn't want to have this here.

@paulfantom

Reply:
I agree with @paulfantom. I commented on your issue @Raynbag I don't think there's any bug in the operator.

Reply:
Ok, I will close this pr first.